src/nvim/option.c

```diff
@@ -908,19 +908,11 @@ static vimoption_T
    SCRIPTID_INIT},
   {"iminsert",    "imi",  P_NUM|P_VI_DEF,
    (char_u *)&p_iminsert, PV_IMI,
-#ifdef B_IMODE_IM
```

Comment:
Hi, I'm multibyte/IM users.

It is for input method control features in GVim.
It seems used for 'imsearch' and 'iminsert' options.
But it is removed in neovim.
Can be removed.